### ISSUE TITLE
Disease List

### ISSUE THREAD

**Issue by yokat** [road86/bahis-data#16]
### Description of feature needing documentation

What are we documenting?The list of diseases coming from old and new bahis have different namings:
https://github.com/road86/bahis-dash/issues/15
The reference, how to deal with the diseases can soon be found in the following document, which is also mentioned in that issue:
https://docs.google.com/spreadsheets/d/1inZYND7AEIboJW6OooHYxv0WWq-YhSV_oYfFpXP-GAc/edit#gid=0

**Comment by yokat** [comment]
I asked Taifur to comment in the spreadsheet in April.

**Comment by ChasNelson1990** [comment]
@taifur-bau can we get an update please?

**Comment by taifur-bau** [comment]
let's discuss it in today's meeting. I want to know further. What is the source of the values coming with braces?

**Comment by yokat** [comment]
The first set with brackets is coming from the old static bahis. The second set without is from the new dynamic bahis.

**Comment by yokat** [comment]
After consulting with @taifur-bau 
* The entries with two or more diseases were from testing in old bahis and should be dropped
* The entries with brackets and stars should be taken without these characters and merged to the new bahis dataset. 
* Leftovers will be communicated back to @taifur-bau

**Comment by yokat** [comment]
There is one empty disease name with 131 incidences from the old bahis. -> dropping, correct?

From the new bahis, all but the following diseases have no correspondence in the old:
"unknown"
"Pneumonia"
"Feline Panleukopenia"
"Classical Swine fever"
Lumpy Skin Disease has a correspondence, but with a slight spelling deviation.

There are 56 diseases in the old bahis, that have no correspondence in the new bahis.

**Comment by taifur-bau** [comment]
Yes, "unknown" and the blank disease values will not be considered for further analysis. Pneumonia, feline panleukopenia, classical swine fever, and lumpy skin disease are newly added diseases in new BAHIS. As lumpy skin disease also was in old BAHIS, so you will need to combine this with new one. For those 56 diseases those are only in old BAHIS, I need to review those first to make any comments regarding these diseases.
